source/common/grpc/common.cc

```diff
@@ -113,7 +115,7 @@ bool Common::resolveServiceAndMethod(const Http::HeaderEntry* path, std::string*
   return true;
 }
 
-Buffer::InstancePtr Common::serializeBody(const Protobuf::Message& message) {
+Buffer::InstancePtr Common::serializeToGrpcFrame(const Protobuf::Message& message) {
```

Comment:
Simplify this one with `serializeMessage` and `prependGrpcFrameHeader`?

Reply:
This results in 2 BufferFragments which is going to be less efficient.  Please confirm and I will make the change.

Reply:
hmm, ok then it is fine, leave a comment?

Reply:
Done